mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown

```diff
@@ -148,6 +148,8 @@ the provider should wait for a single HTTP request.  This will not adjust the
 amount of time the provider will wait for a logical operation - use the resource
 timeout blocks for that.
 
+* `request_reason` - (Optional) Send a Request Reason [System Parameter](https://cloud.google.com/apis/docs/system-parameters) for each API call made by the provider.  The `X-Goog-Request-Reason` header value is used to provide a user-supplied justification into GCP AuditLogs. The value set within the provider is overridden if `CLOUDSDK_CORE_REQUEST_REASON` environment variable is used.
```

Comment:
This should be added under "Full Reference" as well. For consistency with the rest of the file, we should only include the env var there and use text like `Alternatively, this can be specified using the
'CLOUDSDK_CORE_REQUEST_REASON' environment variable.`

Reply:
Provider config will override the environment variable w/ this implementation (which is the correct behaviour imo) so the docs should reflect as much!